### PR TITLE
Add Imports to Tutorial Page

### PR DESCRIPTION
**Pull request by bllchmbrs**
It's confusing when the imports aren't show because it makes it hard to look at the docs as a single artifact. Adding them.

### CODE REVIEW

File: docs/source/tutorial.rst [hunk 1]
```diff
@@ -32,7 +32,18 @@ Training the Model
 First, train a linear regression model that takes two hyperparameters: ``alpha`` and ``l1_ratio``. The code is located at ``example/tutorial/train.py`` and is reproduced below.
 
 .. code:: python
-
+    import os
```
Comment (andrewmchen) [inline docs/source/tutorial.rst]: Do we need the newline here?

File: docs/source/tutorial.rst [hunk 2]
```diff
@@ -32,7 +32,18 @@ Training the Model
...
+    import warnings
```
Comment (mateiz) [inline docs/source/tutorial.rst]: I don't think we use `warnings`, do we?

**Comment by anabranch** [comment]
@mateiz we use warnings here: https://github.com/anabranch/mlflow/blob/patch-1/example/tutorial/train.py#L28

I had just copy-pasted the imports - should be good now.

**Comment by mateiz** [comment]
OK, makes sense but it was just weird because we didn't use them in the tutorial code on that page. I'll merge the updated version.

**Pull request merged**